pom.xml

```diff
@@ -43,8 +43,15 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <!-- TODO remove once vaadin-components-testbench is removed from components -->
```

Comment:
This shouldn't be needed in 14

Reply:
true, sorry for leaving it in originally.